doc/source/examples/dierickx_eccentricities.py

```diff
@@ -0,0 +1,192 @@
+import os, os.path
+import pexpect
```

Comment:
Is this actually used? Doesn't seem to be a standard library module, so users would have to install it.

Reply:
think that got left in there from an earlier version i was tinkering with - can definitely be thrown out!